app/_src/deck/reference/deck_file.md

```diff
@@ -15,14 +15,20 @@ Subcommand to host the decK file manipulation operations.
 
 {% include_cached /md/deck-global-flags.md kong_version=page.kong_version %}
 
-## See also
+## decK file commands
 
-* [deck](/deck/{{page.kong_version}}/reference/deck)	 - Administer your Kong clusters declaratively
 * [deck file add-plugins](/deck/{{page.kong_version}}/reference/deck_file_add-plugins)	 - Add plugins to objects in a decK file
 * [deck file add-tags](/deck/{{page.kong_version}}/reference/deck_file_add-tags)	 - Add tags to objects in a decK file
+{% if_version gte:1.28.x %}
+* [deck file convert](/deck/{{page.kong_version}}/reference/deck_file_convert)	 - Convert files from one format into another format
+* [deck file lint](/deck/{{page.kong_version}}/reference/deck_file_lint)	 - Lint a file against a ruleset
```

Comment:
```suggestion
* [deck file lint](/deck/{{page.kong_version}}/reference/deck_file_lint)	 - Validate a file against a ruleset
```

Reply:
Leaving this one as is, since there's a separate `validate` command.